src/native/corehost/hostmisc/utils.cpp

```diff
@@ -450,18 +450,22 @@ pal::string_t get_download_url(const pal::char_t* framework_name, const pal::cha
         url.append(_X("missing_runtime=true"));
     }
 
+#if defined(TARGET_WINDOWS)
+#define TARGET_OS "win"
+#elif defined(TARGET_OSX)
+#define TARGET_OS "osx"
+#elif defined(TARGET_LINUX) || defined(TARGET_ANDROID)
+#define TARGET_OS "linux"
+#else
+// others are community supported platforms
+#define TARGET_OS "community"
```

Comment:
This is faults everything that is not win/osx/linux into a single bucket. We will get worse usage statistics.

Reply:
AFAIK, the real telemetry is collected by the SDK. If the backend is collecting statistics when the user is navigating to URL from the missing framework error message, I can list the community platforms here as well.

Reply:
there are website telemetry collected by the aka.ms links too. We can probably fix the back-end service to resolve correctly if I know the missing cases and where they should go.

Reply:
but I don't use specific values for telemetry just the source so you can make changes to the parameters if needed. but if we're missing some redirects, I'd love if the customer can get straight to the right download. Is this what drives the aka to show up in the first place?

Reply:
>  if I know the missing cases and where they should go

We can list a few major ones, but the definition of non-portable RID is that anyone can put anything they want in there.

Reply:
These are portable system names independent of PortableBuild. In this context, the missing ones are desktop OS names: FreeBSD, NetBSD, illumos and Solaris.

Currently, the page redirects to `cid=getdotnetcore&arch=..` without `rid=` or `os=`parts which makes sense because there is no Microsoft build for those. That's why I picked up `community`. If it helps the telemetry, I can replace "community" with base system names.

Reply:
the only value I use for telemetry is the cid=getdotnetcore. The rest doesn't matter from an analytics perspective. It matters to the redirection service. We can add new cases if needed.